client/e2e/pages/authenticated/dashboard/profile/page.spec.ts

```diff
@@ -0,0 +1,86 @@
+// 🧪 Suite to test the authentication\profile
+
+import { test } from "@playwright/test";
+// 🪄 Page Object Models
+import { DashboardPOM } from "@/e2e/poms/dashboard";
+import { ProfilePOM } from "@/e2e/poms/profile";
+// ☰ Enums
+import { UserRole } from "@/e2e/utils/enums";
+// 🥞 Connection pool to postgres DB
+import { pool } from "@/e2e/utils/pool";
+import { deleteUserNew, upsertUserIO } from "@/e2e/utils/queries";
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./e2e/.env.local" });
+
+// 🛠️ Function: deletes the new user record from the database
+const deleteNewUserRecord = async () => {
+  try {
+    const query = deleteUserNew;
+    // Execute the deletion query
+    await pool.query(query);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error("Error deleting new user record:", error);
+    throw error;
+  }
+};
+// 📚 Declare a beforeAll hook that is executed once per worker process before all tests.
+// 🥞 Set DB for profile update
+// For industry_user, ensure there is an associated user
+// For no role/new user, ensure there is NOT an associated user
+test.beforeAll(async () => {
+  try {
+    // 👤 industry_user: bc-cas-dev-secondary
+    // Upsert a User record
+    let query = upsertUserIO;
+    // ▶️ Execute the query
+    await pool.query(query);
+
+    // 👤 new user: bc-cas-dev-three
+    // Delete User record
+    await deleteNewUserRecord();
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error("❌ Error in Db setup for profile roles:", error);
+    throw error;
+  }
+});
+
+// 🏷 Annotate test suite as serial
+test.describe.configure({ mode: "serial" });
+test.describe("Test Page - Profile", () => {
+  test.describe(`Test User Role`, () => {
+    // ➰ Loop through the entries of UserRole enum
+    for (let [role, value] of Object.entries(UserRole)) {
+      // No need to test all roles...1 cas, 1 industry_user, and 1 new user should do it
+      if (
+        value === UserRole.CAS_PENDING ||
+        value === UserRole.CAS_ANALYST ||
```

Comment:
Should one of these be the new user?

Reply:
`bc-cas-dev-three` gets into the db during test: `Test Profile Update for none`
Testing  `Test Profile Update` for roles `cas_admin, industry_user, noe (new user)`